msal/application.py

```diff
@@ -280,7 +285,7 @@ def get_authorization_request_url(
             redirect_uri=redirect_uri, state=state, login_hint=login_hint,
             prompt=prompt,
             scope=decorate_scope(scopes, self.client_id),
-            nonce=nonce,
+            nonce=nonce, domain_hint=domain_hint,
```

Comment:
```suggestion
            nonce=nonce,
            domain_hint=domain_hint,
```
Besides the usual "minimizing the amounts of lines being changed" preference, we could actually make this PR contains all "green lines" and not a single "red line", which would become a practical visual clue to convey "this PR is adding a new feature, without needing to change any existing behavior". :-)